tests/Unit/RemoveMemberTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Feature;
+
+use App\Models\Organization;
+use App\Models\User;
+use App\Services\GithubWebhookService;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class RemoveMemberTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testRemoveMember(): void
+    {
+        $user = User::factory()->create(["github_id" => 123]);
+        $organization = Organization::factory()->create(["github_id" => 456]);
+
```

Comment:
```suggestion
```
For me its first step of test - preparing data.

Reply:
moved it to setup

Reply:
if u have one test, u can use `setUp` method but it's not necessary.